public/docs/ts/latest/guide/architecture.jade

```diff
@@ -92,21 +92,19 @@ figure
 
```

Comment:
Above, the `provider` entry needs a " - "

Also, what follows the " - " should be a noun phrase, so the bootstrap entry should be something like this:

   * `bootstrap` - the main application view...